protobufs/livekit_sip.proto

```diff
@@ -270,13 +271,17 @@ message ListSIPTrunkResponse {
 }
 
 message ListSIPInboundTrunkRequest {
+  repeated string trunk_ids = 1;
```

Comment:
Can we add documentation for all these fields, particularly stating that these are optional parameters to filter the response, and that by defaults all entries will be returned?

Reply:
Done!